src/cockpit/misc/print.py

```diff
@@ -50,6 +51,16 @@ def control(self, command: str, **kwargs: Any) -> None:
         """Send a control message, build from **kwargs"""
         self.json('', command=command, **kwargs)
 
+    def authorize(self, cookie: str, b64_content: str, auth_type: str = "Basic", **kwargs: Any) -> None:
```

Comment:
Probably make that just take the "response" as a single string instead of two strings stuck together?

Reply:
We can't use colon due to it being a limited character. Some parsing thing done in the main function

That's why I split it up